arviz/tests/test_diagnostics.py

```diff
@@ -150,7 +152,7 @@ def test_deterministic(self):
         assert (abs(reference["rhat_rank"] - arviz_data["rhat_rank"]) < 6e-5).all(None)
         assert abs(np.median(reference["rhat_rank"] - arviz_data["rhat_rank"]) < 1e-14).all(None)
         not_rhat = [col for col in reference.columns if col != "rhat_rank"]
-        assert (abs(reference[not_rhat] - arviz_data[not_rhat]) < 1e-11).all(None)
+        assert (abs((reference[not_rhat] - arviz_data[not_rhat])).values < 1e-8).all(None)
```

Comment:
Is this just random noise or did the function change?

Reply:
There was a small change :( so I had to adjust the tests.

Reply:
Ok. (The accuracy for rhat is something like 1e-6, so this change is fine)